projects/transfer-journey/src/lib/views/make-transfer-view/make-transfer-view.component.spec.ts

```diff
@@ -0,0 +1,133 @@
+import { CommonModule } from "@angular/common";
+import { Component, EventEmitter, Input, Output } from "@angular/core";
+import { ComponentFixture, fakeAsync, TestBed } from "@angular/core/testing";
+import { By } from "@angular/platform-browser";
+import { ActivatedRoute, Router } from "@angular/router";
+import { of } from "rxjs";
+import { Account } from "transfer-journey";
+import { Transfer } from "../../model/Account";
+import { MakeTransferAccountHttpService } from "../../services/make-transfer-accounts.http.service";
+import { MakeTransferJourneyConfiguration } from "../../services/make-transfer-journey-config.service";
+import { MakeTransferJourneyState } from "../../services/make-transfer-journey-state.service";
+import { MakeTransferPermissionsService } from "../../services/make-transfer-permissions.service";
+import { MakeTransferViewComponent } from "./make-transfer-view.component";
+
+let fixture: ComponentFixture<MakeTransferViewComponent>;
```

Comment:
This is often declared in the describe block, is there any particular reason to avoid it here?

Reply:
moved to the block